core/src/main/java/greencity/dto/errorsresponse/ErrorsResponseDto.java

```diff
@@ -0,0 +1,18 @@
+package greencity.dto.errorsresponse;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class ErrorsResponseDto {
+    boolean status;
```

Comment:
`status` is redundant as it's value is equal to `!errors.isEmpty()`

Reply:
Okay